javascriptv3/example_code/cross-services/lex-bot/README.md

```diff
@@ -0,0 +1,23 @@
+# TypeScript environment for tutorial using Amazon API Gateway to invoke AWS Lambda functions
+This is a workspace where you can find an AWS SDK for JavaScript version 3 (v3) tutorial that how to build and deploy an Amazon Lex Chatbot
```

Comment:
The Lex docs (https://docs.aws.amazon.com/lexv2/latest/dg/what-is.html) show "chatbot".